scalableminds/webknossos
# Implement viewing sharded neuroglancer precomputed datasets

### URL of deployed dev instance (used for testing):
- https://sharding.webknossos.xyz

### Steps to test:
- Sharded datasets to explore and view:
	- gs://h01-release/data/20210601/4nm_raw
	- See [neuroglancer](https://neuroglancer-demo.appspot.com/#!%7B%22dimensions%22:%7B%22x%22:%5B4e-9%2C%22m%22%5D%2C%22y%22:%5B4e-9%2C%22m%22%5D%2C%22z%22:%5B3.3e-8%2C%22m%22%5D%7D%2C%22position%22:%5B499195.875%2C364427.59375%2C2646.5%5D%2C%22crossSectionScale%22:1339.4307643944185%2C%22projectionScale%22:1048576%2C%22layers%22:%5B%7B%22type%22:%22image%22%2C%22source%22:%22precomputed://gs://h01-release/data/20210601/4nm_raw%22%2C%22tab%22:%22source%22%2C%22name%22:%224nm_raw%22%7D%5D%2C%22selectedLayer%22:%7B%22visible%22:true%2C%22layer%22:%224nm_raw%22%7D%2C%22layout%22:%224panel%22%7D) for comparison

### Issues:
- fixes #6823 

------
(Please delete unneeded items, merge only when none are left open)
